### PR TITLE
fix xref's text being double escaped

### PR DESCRIPTION
**Pull request by superyyrrzz**
@vwxyzh @chenkennt @ansyral @hellosnow @qinezh

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs [hunk 1]
```diff
@@ -114,7 +114,6 @@ public HtmlAgilityPack.HtmlNode ConvertToHtmlNode(string language)
                 string value = InnerHtml;
                 if (string.IsNullOrEmpty(value))
                 {
-                    value = StringHelper.HtmlEncode(Text);
                     if (string.IsNullOrEmpty(value) && Spec != null)
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs]: >string.IsNullOrEmpty(value) [](start = 24, length = 27)

string.IsNullOrEmpty(value) must be true?

File: src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs [hunk 2]
```diff
@@ -114,7 +114,6 @@ public HtmlAgilityPack.HtmlNode ConvertToHtmlNode(string language)
                 string value = InnerHtml;
                 if (string.IsNullOrEmpty(value))
                 {
-                    value = StringHelper.HtmlEncode(Text);
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs]: >value [](start = 20, length = 5)

should value fall back to text first? even not encode

Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs]: yes!

---
In reply to: [83991038](https://github.com/dotnet/docfx/pull/799#discussion_r83991038) [](ancestors = 83991038)

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**